redaxo/src/core/pages/setup.php

```diff
@@ -165,10 +165,10 @@
         rex_file::getConfig(rex_path::core('default.config.yml')),
         rex_file::getConfig($configFile)
     );
-}
 
-if ($config['server'] == 'https://www.redaxo.org/') {
-    $config['server'] = 'https://' . $_SERVER['HTTP_HOST'];
+    if (isset($_SERVER['HTTP_HOST']) && $config['server'] == 'https://www.redaxo.org/') {
```

Comment:
ich meine gelesen zu haben dass du angemerkt hast, dass die notice kommt dass `$config` nicht den `server` index hat... falls ich mich nicht täusche sollte diese notice noch immer kommen?

Reply:
> falls ich mich nicht täusche sollte diese notice noch immer kommen?

Nein, da ich das `if` in das vorherige `if` verschoben habe und dort die Variable `$config` initiiert wird. `server`kommt dann aus der `config.yml` und wäre somit gesetzt.